documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+# Consent decision-making
+
+The Openverse maintainers have elected to adopt a decision-making model based on
+the
+[Sociocracy practice of "consent decision-making"](https://patterns.sociocracy30.org/consent-decision-making.html).
+We hope this will accomplish the following:
+
+- Bring expediency to decision-making by giving guidelines for how and what
+  types of feedback to give
```

Comment:
Does "Bring expediency" here mean "speed up", as in "expedite", or "expediency" as in convenience/practicality/suitability? 

Not trying to be pedantic, but if speed is the goal a term like "speed up" might be clearer to contributors.

Reply:
Both but primarily the latter. Speed in and of itself does not seem, to me, a reasonable goal unless we just want to break things. Making expedient decisions, on the other hand, means we approach problem-solving and decision-making with more practicality to avoid getting bogged down in details that may not need to be addressed "_now_" (at the time of the particular decision). In particular this is achieved via the paramount/non-paramount objection distinction.

Please feel free to suggest alternative wording here. I can't think of anything that doesn't accidentally emphasis speed (e.g., "Expedite ...") or make this even wordier.

Reply:
Expediency seems like the perfect word here, then.